projects/github-actions/repo-gardening/src/tasks/gather-support-references/index.js

```diff
@@ -102,9 +105,128 @@ ${ issueReferences
 	.join( '' ) }
 `;
 
+	// If this issue was escalated, make note of it, so next time we edit that comment, we won't escalate it again.
+	if ( needsEscalationNote === true ) {
+		commentBody += `\n*Note: a notification was sent for this issue.*`;
+	}
+
 	return commentBody;
 }
 
+/**
+ * Build an object containing the slack message and its formatting to send to Slack.
+ *
+ * @param {WebhookPayloadIssue} payload - Issue event payload.
+ * @param {string}              channel - Slack channel ID.
+ * @param {string}              message - Basic message (without the formatting).
+ * @returns {Object} Object containing the slack message and its formatting.
+ */
+function formatSlackMessage( payload, channel, message ) {
+	const { issue, repository } = payload;
+	const { html_url, title } = issue;
+
+	let dris = '@bug_herders';
```

Comment:
I wonder if it makes sense to extract the repo <> dri mapping into a config of some sort.

Reply:
On another thought, DRIs are user groups, so the mapping shouldn't change often enough to justify additional complexity.

Reply:
Yeah, on the long term I think we should extract this to a config, for sure, just like we were discussing in #19897. That's on my to-do list, but I'm okay with leaving this hardcoded for now.